### PR TITLE
InfluxDB 2.0 beta 7 release notes

### PR DESCRIPTION
**Pull request by kelseiv**
update per #868

### CODE REVIEW

File: content/v2.0/reference/release-notes/influxdb.md [hunk 1]
```diff
@@ -8,6 +8,32 @@ menu:
...
 ---
 
+## v2.0.0-beta.7 [2020-03-27]
+
+### Features
+
+- Add option to display dashboards in [light mode](/v2.0/visualize-data/dashboards/control-dashboard/#toggle-dark-mode-and-light-mode).
+- Add [shell `completion` commands](/v2.0/reference/cli/influx/completion/) to the `influx` CLI.
+  specified shell (`bash` or `zsh`).
+- Make all `pkg` resources unique by `metadata.name` field.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/influxdb.md]: Scott, for line 18, how did you word this in the package/templates doc and what doc best to link to here?

Comment (sanderson) [reply content/v2.0/reference/release-notes/influxdb.md]: I don't know you need to link anything here. This is regarding the act JSON/YAML file that the `pkg` command outputs which currently we don't document. You could change `pkg` to "template," but there's still some bike-shedding going on here that will likely change the entire nomenclature around templates/packages (there's some cool stuff coming).

Comment (kelseiv) [reply content/v2.0/reference/release-notes/influxdb.md]: Wanted to be sure we didn't have new pkg/templates docs we wanted to introduce for beta 7

File: content/v2.0/reference/release-notes/influxdb.md [hunk 2]
```diff
@@ -8,6 +8,36 @@ menu:
...
+- Ensure Telegraf configuration tokens aren't retrievable after creation. New tokens can be created after Telegraf has been setup.
+- [Delete bucket by name](/v2.0/organizations/buckets/delete-bucket/#delete-a-bucket-by-name) using the `influx` CLI.
+- Add module to write line data to specified url, org, and bucket.
+- Add foundation for pkger stacks, stateful package management.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/influxdb.md]: should "pkger stacks" be "pkg stack"?

Add foundation for package management (`influx pkg`), including stacks and stateful package management?

link to (/v2.0/reference/cli/influx/pkg/)?

File: content/v2.0/reference/release-notes/influxdb.md [hunk 3]
```diff
@@ -8,6 +8,36 @@ menu:
...
+
+### Features
+
+- Add option to display dashboards in [light mode](/v2.0/visualize-data/dashboards/control-dashboard/#toggle-dark-mode-and-light-mode).
+- Add [shell `completion` commands](/v2.0/reference/cli/influx/completion/) to the `influx` CLI.
+  specified shell (`bash` or `zsh`).
+- Make all `pkg` resources unique by `metadata.name` field.
+- Ensure Telegraf configuration tokens aren't retrievable after creation. New tokens can be created after Telegraf has been setup.
+- [Delete bucket by name](/v2.0/organizations/buckets/delete-bucket/#delete-a-bucket-by-name) using the `influx` CLI.
+- Add module to write line data to specified url, org, and bucket.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/influxdb.md]: is module to write line data related to the annotated CSV and linepoint annotation?

File: content/v2.0/reference/release-notes/influxdb.md [hunk 4]
```diff
@@ -8,6 +8,36 @@ menu:
...
+- Add foundation for pkger stacks, stateful package management.
+- Add `--no-tasks` flag to disable scheduling of tasks.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/influxdb.md]: what command was this flag added to?

File: content/v2.0/reference/release-notes/influxdb.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/reference/release-notes/influxdb.md]: I'd just say:

> Added `pkg stack` command to provide stateful package management.

With a link to `/v2.0/reference/cli/influx/pkg/stack`

File: content/v2.0/reference/release-notes/influxdb.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/reference/release-notes/influxdb.md]: No, that isn't related to the annotated CSV change. That's something else that I don't know much about.

File: content/v2.0/reference/release-notes/influxdb.md (hunk 4 above)
Comment (sanderson) [reply content/v2.0/reference/release-notes/influxdb.md]: `influxd`

File: content/v2.0/reference/release-notes/influxdb.md (hunk 3 above)
Comment (hoorayimhelping) [reply content/v2.0/reference/release-notes/influxdb.md]: it's a small helper I wrote to make it easier to write metrics

**Pull request merged**